src/main/ruby/core/enumerable.rb

```diff
@@ -716,8 +716,17 @@ def max_n(n, &block)
   end
```

Comment:
Could you update `max_n` above to also use `last(n).reverse`?

Reply:
Yes, updated `max_n` to `last(n).reverse`